ci/custom_stemcell/tasks/capture-stemcell.sh

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env bash
 
 set -x
```

Comment:
Make sure to check exit code in each script

`set -o errexit -o nounset -o pipefail`